recipes/fasttransform/LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
The license file is in the tarball. You don't have to add it here again.

Reply:
Remove the license.txt file and rely on what comes from the tarball